cypress/integration/accessibility_Iconography and SVGs_spec.js

```diff
@@ -0,0 +1,12 @@
+describe('Iconography and SVGs Accessibility check', () => {
+  beforeEach(() => {
+    cy.visit('/iconography_and_svgs');
+    cy.injectAxe();
+  });
+
+  // Disable the test for now since it's timing out on GitHub CI
```

Comment:
Bummer. Do you have a strategy to get it working?

Reply:
copy pasting bad... cleaning up

Reply:
DONE